### ISSUE TITLE
Accessibility 2026 - Multiple

### ISSUE THREAD

**Issue by MikeDelue** [ua-snap/dash-cc#113]
- Language for page is missing or invalid. Fix by setting page language as "en"
- Form label issues: 5 are orphaned and one is missing. Fixing the orphan to label the missing will probably solve that, the other 4 simply ned to be appropriately labeled to their form inputs:

<img width="1448" height="416" alt="Image" src="https://github.com/user-attachments/assets/1ceef7d8-d1d4-44d5-a8c5-6e84dbb6fc11" />

<img width="1213" height="415" alt="Image" src="https://github.com/user-attachments/assets/95a857c8-2e52-4bb3-aae5-ed76ee43c374" />

- Two skipped heading levels

<img width="1300" height="234" alt="Image" src="https://github.com/user-attachments/assets/af5e3c3d-ca89-4eb7-ade3-e0a3b3ac2904" />

<img width="1548" height="236" alt="Image" src="https://github.com/user-attachments/assets/4cc11bca-acd7-4c7f-bc59-2da6841abf78" />

- Line explaining to "click the camera icon..." fails the contrast test.

<img width="1708" height="188" alt="Image" src="https://github.com/user-attachments/assets/0fc90c36-7601-4977-8886-af263d3db33a" />

- Missing alt text for NWT logo, UAF logo, and map of covered areas.
